Vikta-API-testing-sample/src/test/java/com/test/api/category/CategoryTest.java

```diff
@@ -57,19 +78,234 @@ public void checkCategoryInAllCategoriesList() {
                 .extract()
                 .as(CategoryDTO[].class);
 
-        assertThat("Category wasn't found in all categories list",
+        assertThat(String.format("Category %s wasn't found in all categories list", categoryId),
                 Arrays.asList(categories),
                 hasItem(equalTo(expectedCategory)));
     }
 
+    @Test
+    public void canListAllCategories(){
```

Comment:
in this test, You just get the same list twice and assert that they're equal. Get it once and assert that it's not empty, and that all elements have required fields